### PR TITLE
Automated Continuous Integration & Delivery issue #10

### PR DESCRIPTION
**Pull request by nelsonic**
Expanding & updating this tutorial to the latest "2.0" UI and adding _deployment_! 😉 
+ [x] How to deploy to a Dokku PaaS on DigitalOcean from CircleCI #10 

@SimonLab when you can please review & merge. thanks.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,220 @@
...
+
+e.g: https://circleci.com/gh/nelsonic/circleci-hello-world-nodejs/edit#badges
+![circle-ci-status-badge](https://user-images.githubusercontent.com/194400/41626981-46e652a2-7416-11e8-8e9e-0a03c8fde0a2.png)
+
+> For why this is a good idea, see: https://github.com/dwyl/repo-badges
+
+
+## Part 2 - Continuous _Deployment_
+
+Continuous Deployment (CD) is the perfect compliment
```
Comment (SimonLab) [inline README.md]: `compliment`? Should it be `complement`

Comment (nelsonic) [reply README.md]: image

Thanks for the _early_ review @SimonLab 😉

Comment (SimonLab) [reply README.md]: Thanks for the compliment

**Comment by nelsonic** [comment]
@SimonLab when you can please review & merge. thanks.

**Review by SimonLab — APPROVED** [review]
:tada:

**Comment by nelsonic** [comment]
thanks @SimonLab 😉

**Pull request merged**